.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+## This is a...
+<!-- Check one of the following options with "x" -->
+<pre><code>
```

Comment:
when we leave out the preformatting,then it should be directly rendered as checkboxes by GitHub Markdown, like in

- [x] Finish my changes
- [ ] Push my commits to GitHub
- [ ] Open a pull request